hack/patch-nix-hash.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+nix --log-format raw build ./dev/nix/
```

Comment:
We can also store this as a variable here instead of re-running the same command below. Can just parse the output below instead